src/Action.php

```diff
@@ -97,6 +119,35 @@ public function load($filename = null)
     }
 
     /**
+     * Loads a source file relatively from "actions/". If the filename is not
+     * provided it assumes the action name.
+     *
+     * @param callable $callback The callable to call.
+     * @return \Tao\Action
+     */
+    public function call(callable $callback = null)
+    {
+        if (!$callback) {
+            try {
+                $callback = require $this->getSourcePath(
```

Comment:
It would be a log notifying that a *call* is about to be made, which can be checked to evaluate that it was effectively being made (for debugging purposes).